### ISSUE TITLE
Flag to configure maxSize/minSize when creating cluster/ig

### ISSUE THREAD

**Issue by arikunbotify** [kubernetes/kops#5065]
I'm trying to build automatic creation of K8s Cluster on AWS, using KOps, and with 3 node-IGs - one per AZ. The main purpose is to avoid ASG AZRebalance, without explicitly suspending it.
From what I found in the documentation I can define _--node-count_ on the `kops create clutser`. But that will set both maxSize/minSize to the same value, and adjusting it will require manually editing the cluster configuration.
Can there be a way to set MAx and Min values for the default node IG and any additional IGs from the creation command?

**Comment by zytek** [comment]
You should probably switch to creating clusters from manifests rather via CLI: https://github.com/kubernetes/kops/blob/master/docs/manifests_and_customizing_via_api.md

**Comment by ameena007** [comment]
zytek I do not agree with what you are suggesting. For fully automating the cluster creation process we need this functionality. In a secure environment we do not want a person to login and edit the configuration manually.

**Comment by ameena007** [comment]
+1 for this feature

**Comment by zytek** [comment]
You can automate it, instead of executing kops with many options you just read a yaml file with full cluster definition (which can be templated by external tools).

kops does not and will not support ALL of cluster configuration options via CLI flags as it is not feasible, hence suggestion for using yaml manifests.

**Comment by arikunbotify** [comment]
@zytek When I tried using yaml manifest I got into multiple problems (e.g. if I supply a yaml file for IG with 1 subnet, and the cluster it belongs to has 2, kops will create the ig with all 3 subnets of the cluster ¯\_(ツ)_/¯ not sure if it's me misunderstanding how to use manifests or a bug, but there were many more problems). Anyway, seems to me min/max are very basic values for cluster and ig. if we already have _--node-count_ doesn't it make sense to break it into 2 arguments?
